k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml

```diff
@@ -5,8 +5,10 @@
     "sha256:10e2f4cc76f8f370f754ca2fe6be82eed819a31a5e639820463c671458e4014f": ["v1.16.14"]
     "sha256:11ce05565dac3f7a6562b424828b86ae194a3622592a59574b7bf603496ad1e2": ["v1.18.8"]
     "sha256:288138b199735aa767b042a2bbdc211fb852bf2e5e8368ab45e42fa1ddb0373f": ["v1.19.0-rc.3"]
+    "sha256:2c1be30e12fd4b99b87d36c758e92d6446a43c9d837a8b84c1dce36fd74579e9": ["v1.19.2"]
```

Comment:
@puerco did we change something to the tooling to sort the map (…an ordered map…)? I would love to see the latest tags on top of the list.

Reply:
i did a PR some time ago and Linus made a comment: https://github.com/kubernetes-sigs/k8s-container-image-promoter/pull/251

Reply:
So no one has to jump to another PR...
Manifests created/updated by [`cip-mm`](https://github.com/kubernetes-sigs/k8s-container-image-promoter/tree/master/cmd/cip-mm) are sorted by digest, not tag, as the digest is a unique ID and multiple tags could be assigned to it.

This is working as intended, though the human view is maybe not ideal.
Linus opened an issue for discussion here: https://github.com/kubernetes-sigs/k8s-container-image-promoter/issues/254